hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -1108,6 +1107,92 @@ onOpenChainCloseTx openState newChainState closedSnapshotNumber contestationDead
 
   OpenState{parameters = headParameters, headId, coordinatedHeadState} = openState
 
+-- | Client request to side load confirmed snapshot.
+--
+-- __Transition__: 'OpenState' → 'OpenState'
+onOpenClientSideLoadSnapshot :: IsTx tx => OpenState tx -> ConfirmedSnapshot tx -> Outcome tx
+onOpenClientSideLoadSnapshot openState sideLoadConfirmedSnapshot =
+  case sideLoadConfirmedSnapshot of
+    InitialSnapshot{} ->
+      if currentConfirmedSnapshot == sideLoadConfirmedSnapshot
+        then -- Spec:  ̅S ← snObj(v̂, ŝ, Û, T̂, 𝑈𝛼, 𝑈𝜔)
```

Comment:
Do these additions require spec changes? If so are they already done? I am not sure if you should have spec comments here unless this piece of code has relevant spec parts.

Reply:
Agree.
I added them just for reference but I agree this feature is not part of the normal user journey thus shouldn't be included in our specs.